include/sliding_window/split/get_fly_genome.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+wget http://ftp.flybase.net/genomes/Drosophila_melanogaster/dmel_REFSEQ/fasta/dmel-all-chromosome-r6.32.fasta.gz
+gunzip dmel-all-chromosome-r6.32.fasta.gz
+
+# remove scaffolds
+sed -i -e '/Scaffold/{N;d;}' dmel-all-chromosome-r6.32.fasta
```

Comment:
Sorry I'm very rusty with `sed`, can you explain this?

Do you replace the String `Scaffold` by `N;d;` within the ids?

Reply:
No, I'm actually searching for lines that match pattern `Scaffold`. These are ID lines. I'm then deleting the ID line and the line after it which is the corresponding scaffold sequence. 

The original reference genome contains not only chromosomal sequences but also around 1800 short sequences for mitochondria, scaffolds etc. I thought the short sequences would just be a distraction while testing. 

And just btw: I will move the `get_fly_genome.sh` and example output files to a more suitable place eventually.

Reply:
Ah okay it removes the complete line.